modules/staking/utils_staking_pool.go

```diff
@@ -12,5 +13,35 @@ func (m *Module) GetStakingPool(height int64) (*types.Pool, error) {
 		return nil, fmt.Errorf("error while getting staking pool: %s", err)
 	}
 
-	return types.NewPool(pool.BondedTokens, pool.NotBondedTokens, height), nil
+	validatorsList, err := m.db.GetValidators()
+	if err != nil {
+		return nil, fmt.Errorf("error while getting validators list: %s", err)
+	}
+
+	var unbondingTokens int64
+	for _, validator := range validatorsList {
+		unbondingDelegations, err := m.source.GetUnbondingDelegationsFromValidator(
+			height,
+			validator.GetOperator(),
+			nil,
+		)
+
+		if err != nil {
+			// Do nothing
+			continue
+		}
```

Comment:
If we want to skip the err I think we can ignore the variable declaration here:
```suggestion
		unbondingDelegations, _ := m.source.GetUnbondingDelegationsFromValidator(
			height,
			validator.GetOperator(),
			nil,
		)
```